packages/svelte-materialify/@types/Tabs.d.ts

```diff
@@ -0,0 +1,22 @@
+import { SvelteComponent } from './shared';
+
+export interface TabsProps {
+    TABS?: any;
```

Comment:
I think this one should be deleted?
I just know that [ItemGroup](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/svelte-materialify/%40types/ItemGroup.d.ts) doesn't have their `<script context="module">`-export in types.
TABS would be imported separately (`import Tabs, { TABS } from '...';`)
Don't know how one would provide types for that.

Reply:
const tabs: { class: string, value: any, activeClass: string, disabled: boolean  }[] = [
] would set types for what tabs (not TABS) in tabs.svelte are allowed to be passed down into it

Reply:
I'm not really a typescript user 😄 
If it works and you know what you're doing, alright, otherwise let's wait until someone needs it 👍